README.md

```diff
@@ -375,6 +381,14 @@ Note: the `height` and `width` props have been deprecated.
 
 ## Features
 
+### Duotone
+
+```javascript
+<FontAwesomeIcon icon="coffee" color="blue" secondaryColor="red" secondaryOpacoty={ 0.4 } />
```

Comment:
Quick typo "Opacoty"

Reply:
Ended up fixing this & doing a git rebase fixup... Wasn't worth littering the commit log with yet another commit.